.woodpecker.yml

```diff
@@ -121,3 +117,20 @@ steps:
       - "curl -d'Lemmy-UI-leptos:${CI_COMMIT_TAG} deployed' ntfy.sh/lemmy_drone_ci"
     when:
       event: tag
+
+services:
```

Comment:
Do these services get started up before any of the steps run?

Reply:
Yep they do.